app/graphql/cache.ts

```diff
@@ -79,6 +79,9 @@ export const createCache = () =>
           colorScheme: {
             read: (value) => value ?? "system",
           },
+          countryCode: {
+            read: (value) => value ?? null,
```

Comment:
```suggestion
            read: (value) => value ?? "SV",
```

you could alternatively use "SV" as the default value here, I think that would make CountryCode no longer optional in the type and simplify the code a little bit.

Reply:
changed